llvm/utils/lit/tests/Inputs/shtest-env-positive/env-calls-env.txt

```diff
@@ -1,25 +1,30 @@
-# Check that internal env can call internal env.
+## Tests env command's handling of nested calls, setting, unsetting, and variable retention.
```

Comment:
Based on the file name and what the test cases actually do, I think this comment should say "Tests the behaviour of chaining env commands together." Or something to that effect anyway.